pywbem/__init__.py

```diff
@@ -45,6 +45,7 @@
 from ._recorder import *  # noqa: F403,F401
 from .config import *  # noqa: F403,F401
 from ._statistics import *  # noqa: F403,F401
+from ._logging import *  # noqa: F403,F401
```

Comment:
I am putting the part of this issue tied to logging component into this pr and we will do the rest with the next pr